test/SourceKit/ResolveInfo/Inputs/other.swift

```diff
@@ -0,0 +1,6 @@
+extension C {
```

Comment:
Could you change the tests to have 2-space indenting? We should really automatically format these but we don't have that infrastructure at the moment 😅